### DISCUSSION TITLE
还有其他部署方法吗？

### DISCUSSION THREAD

**Discussion by M3ngZ** [swuecho/chat#358] [Q&A] [answered]
你好，我是个小白，不太懂这些，我有claude的API，我用介绍里的railway部署的，部署完之后，配置了API，可以响应，但经常没有回应，刷新好几次都没用，也不知道是claude自己的原因，还是railway的原因。如果是railway的原因的话，不知道还有没有其他部署方法？

**Comment by swuecho** [comment, marked as answer]
你如果不懂docker ,railway 时最简单的方法.

如果是有时候有问题有时候没响应, 说明你部署是成功的, 可能是程序bug.

你需要尽量复现. 然后把复现的流程截图.

我看一下怎么回事.

你也可以去 https://swuecho-chat-m.fly.dev/ 我这个站点注册一个账号, 这上面能重复出来最好.

**Reply by M3ngZ** [reply]
这几天我一直受这个不回复的折磨。很奇怪的是，今天它突然就好了，说明我部署是没有问题的，可能就是单纯服务器有点毛病吧，谢谢大佬耐心回复，多有打扰。我再学学怎么用docker部署一个吧，我之前学生试用了一个阿里云的服务器还没过期，打算瞎搞一下看看。

**Reply by M3ngZ** [reply]
对了，我还想问一下，我看官网还像是有claude1.3 100k模型，你这个是用的1.0 100K模型吗？不知道能不能跟进一下

**Reply by swuecho** [reply]
默认的1.0 就是1.3 你也可以在在管理界面自己加模型 看看其他的讨论  有相关的话题

你有问题就尝试复现 能复现一般就可以修

**Comment by Sam5440** [comment]
> 你如果不懂docker ,railway 时最简单的方法.
> 
> 如果是有时候有问题有时候没响应, 说明你部署是成功的, 可能是程序bug.
> 
> 你需要尽量复现. 然后把复现的流程截图.
> 
> 我看一下怎么回事.
> 
> 你也可以去 https://swuecho-chat-m.fly.dev/ 我这个站点注册一个账号, 这上面能重复出来最好.
> 
> 
> 
> 

能不能支持vercel部署（？） railway老是封我号（）不知道为啥

**Reply by swuecho** [reply]
vercel 印象中是只能部署前端 edge function 是 js

我这边需要能部署后端 所以可能不太行

**Reply by swuecho** [reply]
封号原因是 bot 不让用 但我的从来没有封过 就是用时会耗玩
